core/src/components/chip/test/shape/chip.e2e.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  /**
+   * This behavior only applies to Ionic Theme.
+   */
+  test.describe(title('chip: shape'), () => {
+    test.describe('default', () => {
+      test('should not have visual regressions', async ({ page }) => {
+        await page.goto(`/src/components/chip/test/shape`, config);
+
+        await page.setIonViewport();
+
+        const container = page.locator('#default');
+
+        await expect(container).toHaveScreenshot(screenshot(`chip-default`));
+      });
+    });
+
+    test.describe('soft', () => {
+      test('should not have visual regressions', async ({ page }) => {
+        await page.goto(`/src/components/chip/test/shape`, config);
+
+        await page.setIonViewport();
```

Comment:
Since we are not taking a screenshot of the entire page this is not needed. Same goes for all the other tests.

Reply:
Removed!